feathr_project/feathr/client.py

```diff
@@ -59,9 +78,10 @@ class FeatureGenerationJobParams:
         feature_config: Path to the features config.
     """
 
-    def __init__(self, generation_config_path, feature_config):
+    def __init__(self, generation_config_path, feature_config, secrets=[]):
```

Comment:
Add docstring for how this secrets will be used

Reply:
Added